.github/workflows/landing-prod.yml

```diff
@@ -0,0 +1,24 @@
+name: Vercel Production Deployment
+env:
+    VERCEL_ORG_ID: ${{ secrets.VERCEL_ORG_ID }}
+    VERCEL_PROJECT_ID: ${{ secrets.VERCEL_PROJECT_ID }}
+on:
+    push:
+        paths:
+            - 'highlight.io/**'
+            - 'docs-content/**'
+        branches:
+            - main
+jobs:
+    Deploy-Preview:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+              with:
+                  submodules: true
+            - name: Install Vercel CLI
+              run: npm install --global vercel@latest
```

Comment:
if you use yarn dlx vercel, no need for the install step here

Reply:
what does that do?

Reply:
https://yarnpkg.com/cli/dlx/

to be consistent with our other usage of yarn, could simplify this so you don't need to install

Reply:
Doesn't it still have to install anyway? I can update for consistency.

I'll have to replace every instance of `vercel` with `yarn dlx vercel` though right?

Reply:
yes that's correct.